components/create-post-page/SetPriceAndDescription.tsx

```diff
@@ -52,7 +52,9 @@ export default function SetPriceAndDescription({
     <FormControl>
       <form onSubmit={handleSubmit}>
         <VStack spacing={7}>
-          <BookCard book={book} isLinkActive={false} />
+          <Box width="128px">
```

Comment:
Do you need to wrap the book card every time? If so, consider doing this in the book card prop

Reply:
Yeah I did it intentionally so that the card would be automatically sized to the grid on other screens so to include them in the create post wizard I just wrapped them in a box. I can add a property that will wrap the card

Reply:
Instead of needing to wrap it at all I just made it so you can specify a width

Reply:
Okay, I see now. The original implementation might be a bit cleaner to work with, but I am fine with either implementation TBH.